regulation-worker/cmd/main.go

```diff
@@ -79,13 +86,18 @@ func Run(ctx context.Context) error {
 	backendconfig.DefaultBackendConfig.WaitForConfig(ctx)
 	identity := backendconfig.DefaultBackendConfig.Identity()
 	dest.Start(ctx)
-
+	oauthV2Enabled := config.GetReloadableBoolVar(false, "RegulationWorker.oauthV2Enabled")
+	// TODO: Remove later
+	pkgLogger.Infon("[regulationApi]", logger.NewBoolField("oauthV2Enabled", oauthV2Enabled.Load()))
+	httpTimeout := config.GetReloadableDurationVar(60, time.Second, "HttpClient.regulationWorker.regulationManager.timeout")
```

Comment:
It doesn't actually make sense to have it. The reason to still go ahead with a reloadable var was in future, if someone decides to do configurations similar to router(at destType), we would have a reloadable var in place already.

Reply:
Let me know if it clarifies or if we want to make changes, it can be done.

Reply:
It would be better to not have a config hotreloadable when it is not. It might confuse people who apply this change and assume that it would be applied automatically without a restart